model-engine/model_engine_server/common/dtos/llms/batch_completion.py

```diff
@@ -243,6 +265,26 @@ class BatchCompletionsJob(BaseModel):
 CreateBatchCompletionsV2Response: TypeAlias = BatchCompletionsJob
 
 
+class UpdateBatchCompletionsV2Request(BaseModel):
+    job_id: str = Field(description="ID of the batch completions job")
+    priority: Optional[int] = Field(
```

Comment:
why would this be useful? wouldn't updating priority require a restart of the job?

Reply:
this would only apply for jobs in queue